### PR TITLE
✨ Frontend: Study home page

### PR DESCRIPTION
**Pull request by odeimaiz**
## What do these changes do?

This PR changes the behavior of the clicking on card action. Now a "Details" or "home page", former "More Options", will be opened.

- [x] Minor improvements to the widget
- [x] e2e tests updated to follow the new workflow

### Bonus
- [x] Rename ``Export`` to ``Export cMIS`` @GitHK 
- [x] Password validator's minimum length increased to 12 @mrnicegyu11 

PorjectHomePage

## Related issue/s

- related to https://github.com/ITISFoundation/osparc-issues/issues/979

### CODE REVIEW

File: services/static-webserver/client/source/class/osparc/auth/core/Utils.js [hunk 1]
```diff
@@ -20,9 +20,9 @@ qx.Class.define("osparc.auth.core.Utils", {
 
   statics: {
     passwordLengthValidator: function(value, item) {
-      const valid = value != null && value.length > 3;
+      const valid = value != null && value.length > 11;
```
Comment (mrnicegyu11) [inline services/static-webserver/client/source/class/osparc/auth/core/Utils.js:23]: beautiful!

**Review by mrnicegyu11 — APPROVED** [review]
I just came here for the security improvement, thanks

File: services/static-webserver/client/source/class/osparc/auth/core/Utils.js (hunk 1 above)
Comment (odeimaiz) [reply services/static-webserver/client/source/class/osparc/auth/core/Utils.js:23]: 💅

Comment (mguidon) [reply services/static-webserver/client/source/class/osparc/auth/core/Utils.js:23]: Should that not go to the backend?

**Review by matusdrobuliak66 — APPROVED** [review]

**Review by mguidon — APPROVED** [review]
:+1:

File: services/static-webserver/client/source/class/osparc/auth/core/Utils.js (hunk 1 above)
Comment (odeimaiz) [reply services/static-webserver/client/source/class/osparc/auth/core/Utils.js:23]: I was about to edit my comment, I guess that the backend also makes the same check...

**Review by SCA-ZMT — APPROVED** [review]

**Review by pcrespov — COMMENTED** [review]
🎉

File: services/static-webserver/client/source/class/osparc/auth/core/Utils.js (hunk 1 above)
Comment (pcrespov) [reply services/static-webserver/client/source/class/osparc/auth/core/Utils.js:23]: I will double check that and adapt as well .

**Review by pcrespov — APPROVED** [review]

**Review by sanderegg — COMMENTED** [review]
nice! somehow we could also show a preview of the workbench in case there are more than 1 node.

**Pull request merged**